pkg/infrastructure/services/resources.go

```diff
@@ -46,12 +46,21 @@ func (s *serviceDeployer) createRequiredResources(instance v1alpha1.KogitoServic
 		resources[reflect.TypeOf(imgv1.ImageStream{})] = []resource.KubernetesResource{imageHandler.imageStream}
 	}
 
+	contentHash, configMap, err := GetAppPropConfigMapContentHash(instance.GetName(), instance.GetNamespace(), s.client)
```

Comment:
When you created the decorator, this will go after the Deployment creation, since is highly dependant on it. If we don't have a image ready yet, won't make sense creating the CM.